cmd/deploy.go

```diff
@@ -49,9 +49,19 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var deployCmd = &cobra.Command{
 		Use:   "deploy",
-		Short: "Build and deploy your Appsody project to your Kubernetes cluster",
-		Long: `This command extracts the code from your project, builds a local Docker image for deployment,
-generates a deployment manifest (yaml) file if one is not present, and uses it to deploy your image to a Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
+		Short: "Build and deploy your Appsody project to Kubernetes.",
+		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
+		
+The command extracts the stack, along with your Appsody project to a local directory. The extracted project is used to build the container image for deployment using 'appsody build'.
+A deployment manifest file, "app-deploy.yaml", is generated if one is not present, and applied to the cluster. The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody Operator, and optionally as a Knative service if you specify the "--knative" flag.
```

Comment:
Can you use steps in Go? I think they would be clearer, eg:

> The command performs the following steps:
> 1. Extracts the stack, along with your Appsody project, to a local directory.
> 2. Runs the `appsody build` command to build the container image for deployment.
> 3. Generates a deployment manifest file, "app-deploy.yaml", if one is not present, then applies it to the cluster. 
> 4. Deploys your image to your Kubernetes cluster via the Appsody operator, and optionally as a Knative service if you specify the "--knative" flag.

Just to check a couple of things:
- Does it makes sense to say a deployment manifest is "applied to a cluster"?
- If you specify --knative, do you get the Knative service **in addition** to the normal deployment? (That's how I interpret the above content.)

Reply:
Steps in Go look good (I've tested and it translates well to the website as well). I would say maybe I could make "then applies it to the cluster" by changing it to "then applies it to your Kubernetes cluster", but it does make sense to use the word "apply" in this case. I will chase up the answer to your second question

Reply:
@doveye it is knative *instead of* normal